### PR TITLE
adding sub categories

### PR DESCRIPTION
**Pull request by elidakirigo**
# 🛠️ Fixes issue #146 
# 📇 Changes proposed 
- added react subcategory
- HTML subcategory
- css subcategory
- backend subcategory
- update to the `_layout/sidebar.md`
- 
# ✅ Check List (Mark all the applicable boxes)

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [ ] This PR contains changes in the documentation
- [x] This PR contains changes in the Code of the project
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by elidakirigo** [comment]
@hasnainmakada-99 let me know if done correctly, I have done the web development section where I am sure of the subcategories,

File: _layouts/sidebar.md [hunk 1]
```diff
@@ -5,6 +5,10 @@
 * Resources
   - [DevOps](./../pages/devops-resources.md)
   - [Web Development](./../pages/web-development-resources.md)
+    - [HTML resources](./../pages/HTML-resources.md)
+    - [CSS resources](./../pages/css-resources.md)
+    - [react](./../pages/react-resources.md)
```
Comment (hasnainmakada-99) [inline _layouts/sidebar.md]: Hey @elidakirigo, please capitalize the 'react' to 'React'

**Review by hasnainmakada-99 — CHANGES_REQUESTED** [review]
@elidakirigo, please do the changes as mentioned 😄 and feel free to ping me here

File: _layouts/sidebar.md (hunk 1 above)
Comment (elidakirigo) [reply _layouts/sidebar.md]: ok

> Hey @elidakirigo, please capitalize the 'react' to 'React'

Comment (elidakirigo) [reply _layouts/sidebar.md]: i have

**Review by hasnainmakada-99 — APPROVED** [review]
Looks Good To Me @elidakirigo, Thanks for your contribution to OSWH, Open Source FTW  🔥

**Pull request merged**